tools/network-scene-vis.md

```diff
@@ -0,0 +1,50 @@
+---
+id: netscenevis
+title: Network Scene Visualization
+---
+
+The Network Scene Visualization (NetSceneVis) is a powerful, user-friendly tool included in the Multiplayer Tools package to help you visualize and debug network communication on a per-object basis in the scene view of your project with visualizations such as mesh shading and text overlay.
+
+The NetSceneVis tool listens to network messages about the game state to visually display bandwidth and ownership on a per-object basis in the Unity Editor. The visualizations are updated in real-time as the network messages are sent and received, showing the game state synchronization across the network.
+
+This tool can help optimize your network code to ensure that your game runs smoothly for all players.
+
+## Requirements
+- Unity 2023.2.0a11 or higher
+- Up-to-date Multiplayer Tools package, see [Install the Multiplayer Tools Package](./install-tools.md)
```

Comment:
We should maybe mention here that the tool is only available in MP Tools >= 2.0.0-pre-3, just so people know the specific version it's available in.